deploy/crds/nodemaintenance_crd.yaml

```diff
@@ -64,6 +64,7 @@ spec:
               format: int64
               type: integer
           type: object
+      type: object
```

Comment:
Is it necessary that `type` is the last line of an element?
Seems like it might be better after line 16, where it's clearer which earlier elements it refers to.

Reply:
tried to move it, but build failed with the following error:

```
./build/make-csv-generator.sh

time="2020-03-31T01:26:09Z" level=info msg="Generating CSV manifest version 9999.9999.9999"

Error: catalog scaffold failed: (deploy/crds/nodemaintenance_crd.yaml: error converting YAML to JSON: yaml: line 19: mapping values are not allowed in this context)
```

Reply:
thats an odd thing for it to enforce